HW2/P3/P3.txt

```diff
@@ -0,0 +1,25 @@
+As shown below, the performance of implementing AVX yields much better time efficiency. And the more threads, the faster.
+1) nonAVX -> AVX_1: 706%
+2) AVX_1 -> AVX_2: 208%
+3) AVX_2 -> AVX_4: 142%
+4) AVX_4 -> AVX_8: 110%
```

Comment:
Nice, I got the same trends as you I believe, but I structured my answer to contrast the performance in number of seconds for a particular # of threads with and without instruction-level parallelism. Here's an example of a result I got (I think they should be close to yours?): 
For example: 2 Threads WITHOUT was 2.71843004227 for me, and 2 threads WITH was 0.374233961105